api/track.js

```diff
@@ -36,7 +36,11 @@ module.exports = async (req, res) => {
     }
   }
 
-  if (hash !== secret && kind !== "impression") {
+  if (
+    hash !== secret &&
+    kind !== "impression" &&
+    kind !== "sponsored_lots_impression"
```

Comment:
Maybe use kind.includes('impression') instead in order to not needing to change if used elsewhere in the future.

Reply:
Updated it to work with a sponored-flag instead and requested review on that and reverted this change.